core/server/services/mega/mega.js

```diff
@@ -213,10 +196,21 @@ const addEmail = async (postModel, options) => {
     const knexOptions = _.pick(options, ['transacting', 'forUpdate']);
     const filterOptions = {...knexOptions, limit: 1};
 
+    // TODO: this is a hack for https://github.com/TryGhost/Team/issues/1626
+    delete postModel.relations.newsletter;
+
     const newsletter = await postModel.related('newsletter').fetch({require: true, ..._.pick(options, ['transacting'])});
 
+    if (newsletter.get('status') !== 'active') {
+        // A post might have been scheduled to an archived newsletter.
+        // Don't send it (people can't unsubscribe any longer).
```

Comment:
What happens to the post in this situation? I assume retrying the send would fail again, but is there a path to update the newsletter and resend?

Reply:
Well, no email will get attached to the post. So they can retry by reverting to draft and publishing again with a different newsletter